feature/bgp/policybase/ate_tests/route_installation_test/route_installation_test.go

```diff
@@ -127,6 +128,18 @@ func configureDUT(t *testing.T, dut *ondatra.DUTDevice) {
 	dc.Interface(i2.GetName()).Replace(t, i2)
 }
 
+//Configure network instance
```

Comment:
Please ensure there is a whitespace after `//`. Also, comments should follow godoc convention.

https://go.dev/blog/godoc

```
// configureNetworkInstance configures the network instance.
```

Actually, in this case the comment doesn't really add new information, so let's just remove the comment.